package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "blob.js": "^1.0.1",
-    "blueimp-file-upload": "blueimp/jQuery-File-Upload#^9.22.1",
+    "blueimp-file-upload": "blueimp/jQuery-File-Upload#~8.1.0",
```

Comment:
oops, downgrading this, but I assume you'll remove this before final merge, thanks!